storage/storage-interface/src/lib.rs

```diff
@@ -664,11 +664,11 @@ pub trait DbWriter: Send + Sync {
         unimplemented!()
     }
 
-    /// Persists merklized states as authenticated state snapshot.
-    /// See [`AptosDB::save_state_snapshot`].
+    /// Persists merklized states as authenticated state checkpoint.
+    /// See [`AptosDB::save_state_checkpoint`].
     ///
-    /// [`AptosDB::save_state_snapshot`]: ../aptosdb/struct.AptosDB.html#method.save_state_snapshot
-    fn save_state_snapshot(
+    /// [`AptosDB::save_state_checkpoint`]: ../aptosdb/struct.AptosDB.html#method.save_state_checkpoint
+    fn save_state_checkpoint(
```

Comment:
I'm actually okay with calling it a snapshot. Let's keep it consistent with your `xDelta` struct at least.